### PR TITLE
Howto apply the special report tone to a tag

### PR DESCRIPTION
**Pull request by DiegoVazquezNanini**
## What does this change?
Adds documentation on how to apply the special report tone to a tag.

@guardian/guardian-frontend

### CODE REVIEW

File: docs/03-dev-howtos/18-apply-the-special-report-tone.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+m = hashlib.md5()
+m.update("a-public-salt3W#ywHav!p+?r+W2$E6="+tag)
+print m.hexdigest()
+```
+3. Add the generated hash to the facia-scala-client on this file and raise a PR against master
+https://github.com/guardian/facia-scala-client/blame/48f90b6173c6409507a58e8046868f0d9e93ca06/fapi-client/src/main/scala/com/gu/facia/api/utils/CardStyle.scala#L52
+
+5. Merge the PR and release the library to Maven using sbt.
+
+6. Update facia-scala-client library on Frontend and MAPI. Merge adn deploy the PRs.
```
Comment (MatthewJWalls) [inline docs/03-dev-howtos/18-apply-the-special-report-tone.md]: adn => and

**Review by jfsoul — APPROVED** [review]
The doc itself looks good, but need to get that TOC working before merging

File: docs/03-dev-howtos/18-apply-the-special-report-tone.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Apply the special report tone
+
+Sometimes CP or someone from Editorial will ask us to apply the special report treatment to a tag.
+The following doc explains how to make that happen.
+
+1. Find out the id of the tag where we need the special report tone. For example "news/series/panama-papers".
+2. Replace <NEW-SPECIAL-TAG> with the string of the special tag and run this code on a python REPL:
+```
```
Comment (MatthewJWalls) [inline docs/03-dev-howtos/18-apply-the-special-report-tone.md]: It wasn't immediately obvious to me that "the id of the tag" and "the string of the special tag" meant the same thing. Perhaps use the same wording in both cases?

**Review by MatthewJWalls — APPROVED** [review]

**Comment by TBonnin** [comment]
That looks quite complicated ⚛️ 
Why is not coming from CAPI?

**Pull request merged**